### PR TITLE
Add logging to record when a user's access token is missing claims

### PR DESCRIPTION
**Pull request by sbryzak**
This PR will log details of a user when their JWT token is missing either the user_id or account_id claims.

Fixes https://issues.redhat.com/browse/CRT-1804

### CODE REVIEW

File: pkg/middleware/middleware.go [hunk 1]
```diff
@@ -66,6 +67,12 @@ func (m *JWTMiddleware) HandlerFunc() gin.HandlerFunc {
 			m.respondWithError(c, http.StatusUnauthorized, err.Error())
 			return
 		}
+
+		if token.UserID == "" || token.AccountID == "" {
+			logrus.Infof("Missing essential claims from token - [user_id:%s][account_id:%s] for user [%s], sub [%s]",
```
Comment (alexeykazakov) [inline pkg/middleware/middleware.go:72]: Can you use our logger instead? Same as you use in signup_service.go

**Review by alexeykazakov — COMMENTED** [review]
Looks good overall but can we use our logger instead?

File: pkg/middleware/middleware.go (hunk 1 above)
Comment (sbryzak) [reply pkg/middleware/middleware.go]: no problem, fixed

**Review by alexeykazakov — APPROVED** [review]

**Review by MatousJobanek — APPROVED** [review]

**Comment by MatousJobanek** [comment]
@xcoulon could you please help with creating a query for Loki to filter/select these logs?

**Pull request merged**